app/controllers/comments_controller.rb

```diff
@@ -1,5 +1,26 @@
 class CommentsController < ApplicationController
-  def index; end
+  def new
+    @comment = Comment.new
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+  end
 
-  def show; end
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.author_id = current_user.id
```

Comment:
- [ ] You have a typo that is bringing an error message when I try to create a comment. Instead of `author_id`, it should be `user_id` .
![image](https://github.com/Microverse-Fullstack-Program/rails-blog-app/assets/45460192/c273f89f-fd3b-4dee-b5d8-7176c628f3d7)